### PR TITLE
Migrate to new s2s bridge

### PR DESCRIPTION
**Pull request by fewensa**
Related: https://github.com/darwinia-network/bridger/pull/479

### CODE REVIEW

File: modules/fee-market/src/lib.rs [hunk 1]
```diff
@@ -255,7 +254,6 @@ pub mod pallet {
 		/// Update locked collateral for enrolled relayer, only supporting lock more. (Update market
 		/// needed)
 		#[pallet::weight(<T as Config<I>>::WeightInfo::update_locked_collateral())]
-		#[transactional]
```
Comment (HackFisher) [inline modules/fee-market/src/lib.rs:258]: why remove this?

File: modules/fee-market/src/s2s/payment.rs [hunk 2]
```diff
@@ -112,7 +112,7 @@ where
 		// Pay treasury_sum reward
 		do_reward::<T, I>(
 			relayer_fund_account,
-			&T::TreasuryPalletId::get().into_account(),
+			&T::TreasuryPalletId::get().into_account_truncating(),
```
Comment (HackFisher) [inline modules/fee-market/src/s2s/payment.rs:115]: what't the difference?

File: bin/runtime-common/src/lib.rs [hunk 3]
```diff
@@ -19,7 +19,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 pub mod messages;
-pub mod messages_api;
 pub mod messages_benchmarking;
 
 #[cfg(feature = "integrity-test")]
```
Comment (AsceticBear) [inline bin/runtime-common/src/lib.rs:24]: Do we still need `integrity` mod?

File: modules/fee-market/src/lib.rs (hunk 1 above)
Comment (AsceticBear) [reply modules/fee-market/src/lib.rs:258]: It's the default behavior in the latest substrate, See https://github.com/paritytech/substrate/pull/11546

File: modules/fee-market/src/s2s/payment.rs (hunk 2 above)
Comment (AsceticBear) [reply modules/fee-market/src/s2s/payment.rs:115]: See https://github.com/paritytech/substrate/pull/10719

File: bin/runtime-common/src/lib.rs (hunk 3 above)
Comment (fewensa) [reply bin/runtime-common/src/lib.rs:24]: It's really useless for us. here is to define some test common methods to test on the chain.
https://github.com/paritytech/parity-bridges-common/blob/e2e283027294a856663a6f4ad5472808c5a24d49/bin/millau/runtime/Cargo.toml#L78

Comment (AsceticBear) [reply bin/runtime-common/src/lib.rs:24]: Then make a clean about this crate?

Comment (fewensa) [reply bin/runtime-common/src/lib.rs]: https://github.com/darwinia-network/darwinia-messages-substrate/pull/149/commits/dd487ecb39be16cd567f074e15bab55c6064a30e

**Review by HackFisher — APPROVED** [review]

**Pull request merged**